response.proto

```diff
@@ -3,11 +3,20 @@ import "type.proto";
 
 package pbnavitia;
 
+enum PathType {
```

Comment:
Is it just for cycles? In this case CyclePathType could be a better name.

Reply:
For now we want to use it only for cycles. BUT I am sure we will use other types for the other modes.